src/pybind/mgr/dashboard/frontend/src/app/shared/components/card/card.component.html

```diff
@@ -1,7 +1,18 @@
-<div class="card shadow-sm flex-fill">
-  <h4 class="card-title mt-4 ms-4 mb-0">
-    {{ cardTitle }}
+<div class="card flex-fill"
```

Comment:
i just noticed that this is removing shadows from everywhere (including the new landing page) which is not good and an unintended behavior. You should remove the shadow whenever you don't need it and not from the whole dashboard.

```suggestion
<div class="card flex-fill"
```

Reply:
Done